### PR TITLE
Remove define hook from documentation

### PR DESCRIPTION
**Pull request by ivanbacher**
# Pull Request

## 📖 Description

In the latest beta release you mention this:
 - refactor(controller): remove define lifecycle hook by @bigopon in https://github.com/aurelia/aurelia/pull/1899

This pull request just removes any mention of define from the component lifecycle hook documentation

### CODE REVIEW

File: docs/user-docs/TOC.md [hunk 1]
```diff
@@ -43,7 +43,7 @@
 * [Slotted content](components/shadow-dom-and-slots.md)
 * [Scope and context](components/scope-and-binding-context.md)
 * [CustomElement API](components/customelement-api.md)
-* Template compilation
+* [Template compilation](components/template-compilation/README.md)
```
Comment (Vheissu) [inline docs/user-docs/TOC.md:46]: We don't want to introduce a new page for the top level and let Gitbook create the index for us instead.

File: docs/user-docs/TOC.md [hunk 2]
```diff
@@ -53,8 +53,8 @@
 
 ## Getting to know Aurelia
 
-* Routing
-  * @aurelia/router
+* [Routing](getting-to-know-aurelia/routing/README.md)
+  * [@aurelia/router](getting-to-know-aurelia/routing/aurelia-router/README.md)
```
Comment (Vheissu) [inline docs/user-docs/TOC.md:57]: We don't want to introduce new pages for the top level and let Gitbook create the index for us instead.

File: docs/user-docs/TOC.md [hunk 3]
```diff
@@ -65,7 +65,7 @@
     * [Route Events](routing/route-events.md)
     * [Router Tutorial](routing/router-tutorial.md)
     * [Router Recipes](routing/router-recipes.md)
-  * @aurelia/router-lite
+  * [@aurelia/router-lite](getting-to-know-aurelia/routing/aurelia-router-lite/README.md)
```
Comment (Vheissu) [inline docs/user-docs/TOC.md:68]: We don't want to introduce a new page for the top level and let Gitbook create the index for us instead.

File: docs/user-docs/components/template-compilation/README.md
Comment (Vheissu) [inline docs/user-docs/components/template-compilation/README.md:1]: Not needed, as per feedback above in the TOC.

File: docs/user-docs/getting-to-know-aurelia/routing/README.md
Comment (Vheissu) [inline docs/user-docs/getting-to-know-aurelia/routing/README.md:1]: Not needed, as per feedback above in the TOC.

File: docs/user-docs/getting-to-know-aurelia/routing/aurelia-router-lite/README.md
Comment (Vheissu) [inline docs/user-docs/getting-to-know-aurelia/routing/aurelia-router-lite/README.md:1]: Not needed, as per feedback above in the TOC.

File: docs/user-docs/getting-to-know-aurelia/routing/aurelia-router/README.md
Comment (Vheissu) [inline docs/user-docs/getting-to-know-aurelia/routing/aurelia-router/README.md:1]: Not needed, as per feedback above in the TOC.

**Review by Vheissu — COMMENTED** [review]
Thank you very much for this @ivanbacher just a few C's.

**Comment by ivanbacher** [comment]
No problem. Ups, not sure what happened there, might have been during the GitBook import/sync. Anyway I removed the unnecessary files

**Comment by bigopon** [comment]
Thanks @ivanbacher , nice work!

**Pull request merged**